### ISSUE TITLE
How can I track the performance of my GPT-2 model during finetuning?

### ISSUE THREAD

**Issue by hmdgit** [huggingface/transformers#3644]
Hi,

I am new in using Transformer HugginFace library. 

I am using [Google Colab](https://colab.research.google.com/github/interactive-fiction-class/interactive-fiction-class.github.io/blob/master/homeworks/language-model/hw4_transformer.ipynb) to fine tune GPT-2 model. Google Colab only displays the output of last 5000 lines. So, I could not be able to figure out the performance of previous checkpoints, whose output vanishes.

I would like to track the performance of my training model during the whole period of time. Is it possible to track it by using **tensorboard**? or is there any other way exist? I have noticed a variable "logging_steps" in [run_language_modeling.py](https://github.com/huggingface/transformers/blob/master/examples/run_language_modeling.py) file, but I donot have an idea how can I use it to track performance of training model in Google Colab?

**Comment by mariamabarham** [comment]
[run_language_modeling.py](https://github.com/huggingface/transformers/blob/master/examples/run_language_modeling.py) allow you to track you training performance (validation perplexity, training loss and learning rate) using tensorboad. You need to have tensorboad installed. And then you can just run `tensorboard --logdir=runs` to follow your training.

**Comment by hmdgit** [comment]
Thanks a lot. It works. 
I have used these commands in Google Colab
```
%load_ext tensorboard
%tensorboard --logdir=runs
```
